src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php

```diff
@@ -1314,15 +1314,15 @@ public function testPassIdAndNameToView()
         $this->assertEquals('name', $view->vars['full_name']);
     }
 
-    public function testStripLeadingUnderscoresAndDigitsFromId()
+    public function testAllowLeadingUnderscoresAndDigitsFromId()
```

Comment:
I'd suggest dropping the test case on branch 5.4 since it doesn't do anything now

Reply:
I've opened a new PR https://github.com/symfony/symfony/pull/54329

Reply:
I've reverted the changes and will rebase the PR once https://github.com/symfony/symfony/pull/54329 is merged.